### PR TITLE
updated readme

### LINKED ISSUE #50: Add localhost run example
## Description:
- Add an example line showing the docker flag to run webv against host on local machine
- The example would add "--net=host" for one example
- Example:
sudo docker run --net=host -it --rm retaildevcrew/webvalidate --host localhost:8080 --filebaseline.json

### PR DESCRIPTION
**Pull request by jofultz**
## Type of PR

- [X] Documentation changes
- [x] Code changes
- [x] Test changes
- [ ] CI-CD changes
- [ ] GitHub Template changes

## Purpose of PR
- Add command to show illustrate running against the localhost
- update readme to use new System.CommandLine params
- fix bug running against localhost

## Issues Closed or Referenced

- Closes #50

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,6 +29,9 @@ docker run -it --rm retaildevcrew/webvalidate --host https://froyo.azurewebsites
 # long running benchmark test
 docker run -it --rm retaildevcrew/webvalidate --host https://froyo.azurewebsites.net --files benchmark.json
 
+# run baseline tests against localhost
+docker run --net=host -it --rm retaildevcrew/webvalidate --host localhost:8080 --filebaseline.json
```
Comment (bartr) [inline README.md:33]: the localhost port should be 4120
missing a space after --file and should be --files (or -f)
is the --net=host pre-defined?

File: README.md [hunk 2]
```diff
@@ -29,6 +29,9 @@ docker run -it --rm retaildevcrew/webvalidate --host https://froyo.azurewebsites
 # long running benchmark test
 docker run -it --rm retaildevcrew/webvalidate --host https://froyo.azurewebsites.net --files benchmark.json
```
Comment (aflinchb) [inline README.md:30]: all of these should be --server now, right?

File: README.md (hunk 1 above)
Comment (jofultz) [reply README.md]: Updated to:
docker run --net=host -it --rm retaildevcrew/webvalidate --server localhost:4120 --files baseline.json

File: README.md (hunk 2 above)
Comment (bartr) [reply README.md:30]: yes - the updates I made to readme for system.commandline didn't get checked in, so I'll update this PR

Comment (bartr) [reply README.md]: PR is updated

**Pull request merged**